src/characteristic/abstractCharacteristic.ts

```diff
@@ -134,4 +169,87 @@ export default abstract class AbstractCharacteristic<
       ...parameters,
     )
   }
+
+  deviceUsesFahrenheit(): boolean {
+    return this.device.getDevice().countryCode.startsWith('US')
```

Comment:
I'm a bit confused -- is this necessary? My understanding was both the LG and HomeKit APIs operate in Celsius, and it's just a matter of what HomeKit/LG display *on screen* that differs

Reply:
Is it necessary? Can't say with units that display celsius. So this was my attempt to limit my changes to only Fahrenheit units, and if we find it's needed with celsius units, we can change. But somebody else would have to test that (someone who has a celsius unit)